projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -974,6 +984,100 @@ function wpcom_launchpad_is_sensei_setup_visible() {
 	return is_plugin_active( 'sensei-lms/sensei-lms.php' );
 }
 
+/**
+ * Determines whether or not the verify domain email task should be visible.
+ *
+ * @return bool True if the verify domain email task should be visible.
+ */
+function wpcom_launchpad_is_verify_domain_email_visible() {
+	// If the task is complete, we should show it and prevent the logic below
+	// to be executed.
+	if ( wpcom_is_checklist_task_complete( 'verify_domain_email' ) ) {
+		return true;
+	}
+
+	$domains_pending_icann_verification = array();
+
+	// For Atomic sites we need to get the domain list from
+	// the public API.
+	if ( ! defined( 'IS_WPCOM' ) || ! IS_WPCOM ) {
```

Comment:
Looks like in a [recent previous PR](https://github.com/Automattic/jetpack/commit/36ccf724d51b7f203111c0a515bf94fa43efed40#diff-3e92c2cff4e2dd4ec2e0ef889cac1c73a78275f711d0cd9b4e419b7753de513eR971) this check for AT siteness was done differently. What made you change? I like the approach from the linked PR because it's a positive check, but I don't know the context, so it might not be the correct way to handle it here

Reply:
Nice catch, thanks!